internal/cloud/backend.go

```diff
@@ -152,8 +152,13 @@ func (b *Cloud) PrepareConfig(obj cty.Value) (cty.Value, tfdiags.Diagnostics) {
 		return obj, diags
 	}
 
+	// check if organization is specified in the config.
 	if val := obj.GetAttr("organization"); val.IsNull() || val.AsString() == "" {
-		diags = diags.Append(invalidOrganizationConfigMissingValue)
+		// organization is specified in the config but is invalid, so
+		// we'll fallback on TF_ORGANIZATION
+		if val := os.Getenv("TF_ORGANIZATION"); val == "" {
+			diags = diags.Append(invalidOrganizationConfigMissingValue)
```

Comment:
This error contains an unevaluated format string, resulting in the following incorrect output:

<img width="857" alt="init" src="https://user-images.githubusercontent.com/68917/160388673-c6fa85a6-5a5a-47d6-a9f3-57c8f9fa38ac.png">